<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>392</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>demoVideo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="video">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>272</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>video</string>
    </property>
   </widget>
   <widget class="QLabel" name="video1">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>480</width>
      <height>272</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>video1</string>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>490</y>
      <width>156</width>
      <height>23</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>481</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="loseLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>模拟丢包率：</string>
     </property>
    </widget>
    <widget class="QSlider" name="loseSlider">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>160</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="loseNum">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>35</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0%</string>
     </property>
    </widget>
    <widget class="QPushButton" name="losePushButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>100</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
    <widget class="QLabel" name="decodeLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Raptor解码成功率：</string>
     </property>
    </widget>
    <widget class="QLabel" name="decodeRate">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>35</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>100%</string>
     </property>
    </widget>
    <widget class="QLabel" name="relayLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>中继节点总数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="relayNum">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QLabel" name="afLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>放大中继编号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="afNum">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>1,3,6</string>
     </property>
    </widget>
    <widget class="QLabel" name="dfLabel">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>译码中继编号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="deNum">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>2,4,5</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>280</y>
      <width>480</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="loseLabel1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>模拟丢包率：</string>
     </property>
    </widget>
    <widget class="QSlider" name="loseSlider1">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>20</y>
       <width>160</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="loseNum1">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>20</y>
       <width>35</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0%</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>479</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
